.htaccess

```diff
@@ -114,7 +127,7 @@ RedirectMatch "^/(?!iphone-and-ipad)(ios|iphone|ipad)(\/.*)?" "/iphone-and-ipad$
 # Connect With Art landing page
 RedirectMatch "/connectwithart(\/|$)" "https://sites.google.com/sil.org/connectwithart/home"
 
-# per-language landing pages
+# per-language landing pages (maltese and rawang)
```

Comment:
Are the initial slashes appropriate?

Reply:
The RedirectMatches wouldn't work with just `^foo`